src/containers/DeveloperPortal/DocumentationSitemap/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import {A, Container} from 'components';
+import './DocumentationSitemap.scss';
+
+// TODO: update links when ready
+export default function DocumentationSitemap() {
+  return (
+    <div className="DocumentationSitemap">
+      <Container className="DocumentationSitemap__container">
+        <div className="DocumentationSitemap__title">Documentation Sitemap</div>
+        <div className="DocumentationSitemap__topics">
+          <div className="DocumentationSitemap__topic-container">
+            <div className="DocumentationSitemap__topic-header">Living Whitepaper</div>
+            <A href="/" newWindow={false} className="DocumentationSitemap__link">
+              Principal Entities on the Network
+            </A>
```

Comment:
any reason you used `<A>` instead of `<Link>` for these? I think these should be `<Link>`, unless you had a good reason. Other than that, this PR is good to go!

Reply:
btw i added a minor commit, so be sure to `git pull` if you are going to make changes

Reply:
oh i used `<A>` just because it's our custom component lol, and it has the new window icon to tell for external links, let me change back to `<Link>` then

Reply:
Changed, ready to go

Reply:
i think i might be tripping, but is there a performance hit for a SPA if we use <a> tags for internal links rather than using `react-router`'s <Link>?